### DISCUSSION TITLE
Reference for RNA character codes recommend different codeset

### DISCUSSION THREAD

**Discussion by ahwagner** [HGVSnomenclature/hgvs-nomenclature#202] [Q&A]
Our recommendation for character codes in RNA sequences is lower-case `acgu`. However, the referenced source of this, the [IUPAC-IUBMB assigned nucleotide symbols](http://www.qmul.ac.uk/sbcs/iubmb/misc/naseq.html#500) recommends to use `ACGT`:

>Thus it is important to note from the outset that the recommended symbols will not discriminate between DNA and RNA, and the symbol T will be employed at all positions where U might appear in the RNA. Similarly, no distinction will be made in the symbols between base, nucleoside and nucleotide. Sequences may be assumed to have a deoxyribose backbone (DNA) unless specified otherwise. These changes from earlier recommendations [[1]](https://iubmb.qmul.ac.uk/misc/naseq.html#01) reflect great advances in techniques for sequencing DNA, so that RNA sequences are now commonly deduced from the corresponding DNA sequences. Since the standard representation of a DNA sequence may be converted to the corresponding RNA sequence by the simple expedient of substituting T by U, it is not envisaged that data banks based on computer storage facilities will inevitably contain entries for both DNA and its RNA equivalent. Authors should always, however, make it clear which strand of DNA or RNA a given sequence refers to, and in circumstances where confusion between DNA and RNA is likely the sequence may be prefixed with the lower-case letter d or r, as in the previous recommendations [[1]](https://iubmb.qmul.ac.uk/misc/naseq.html#01).

It appears that HGVS nucleotide character conventions do not follow these referenced recommendations–is this in error? Or am I missing something?

**Comment by reece** [comment]
I agree with your reading, @ahwagner. I don't have an suggestion for how to handle this.

**Reply by jtdendunnen** [reply]
Indeed, you are right, excellent you picked this up. I suggest we make the required changes in the recommendations as soon as possible. It also solves the problems I had with Issue#147.
